build.gradle

```diff
@@ -1129,7 +1129,7 @@ project(':core') {
   if (userEnableTestCoverage) {
     scoverage {
       scoverageVersion = versions.scoverage
-      scoverageScalaVersion = '2.13.16' // `org.scoverage:scalac-scoverage-plugin_2.13.16:2.3.0` is the latest as of now
+      scoverageScalaVersion = versions.scala // `org.scoverage:scalac-scoverage-plugin_2.13.17:2.4.0` is the latest as of now
```

Comment:
it seems the comment is unnecessary. Could you please remove it?

Reply:
Done.